tendrl/gluster_integration/gdeploy_wrapper/plugins/gdeploy.py

```diff
@@ -0,0 +1,202 @@
+import logging
+
+from tendrl.node_agent.provisioner.gluster.provisioner_base import\
+    ProvisionerBasePlugin
+try:
+    import python_gdeploy.actions
+except ImportError:
+    raise ImportError("This node does not have python-gdeploy installed")
+
+from python_gdeploy.actions import create_gluster_volume
+from python_gdeploy.actions import delete_volume
+from python_gdeploy.actions import start_volume
+from python_gdeploy.actions import stop_volume
+from python_gdeploy.actions import rebalance_volume
+from python_gdeploy.actions import expand_gluster_volume
+from python_gdeploy.actions import shrink_gluster_volume
+
+LOG = logging.getLogger(__name__)
+
+
+class GdeployPlugin(ProvisionerBasePlugin):
+    def __init__(self):
+        ProvisionerBasePlugin.__init__(self)
+
+    def create_volume(self, volume_name, brick_details, transport=None,
+                      replica_count=None, disperse_count=None,
+                      redundancy_count=None, tuned_profile=None, force=False):
+        args = {}
+        if trasnport:
+            args.update({"transport":transport})
+        if replica_count:
+            args.update({"replica_count": replica_count})
+        if disperse_count:
+            args.update({"disperse_count": disperse_count})
+        if redundancy_count:
+            args.update({"redundancy_count": redundacy_count})
+        if tuned_profile:
+            args.update({"tuned_profile": tuned_profile})
+        if force:
+            args.update({"force":force})
+
+        out, err, rc = create_gluster_volume.create_volume(
+            volume_name,
+            brick_details,
+            **args
+        )
+        if rc == 0:
+            LOG.info("gluster volume created successfully")
+        else:
+            LOG.error(
+                "gluster volume creation failed. Details %s" % str(out)
+            )
+            return False
+        return True
+
+    def delete_volume(self, volume_name, host=None, force=None,
+                      clear_bricks=None):
+        args = {}
+        if host:
+            args.update({"host":host})
```

Comment:
Is this `host` required for targeting the node on which the command should be run? Otherwise what is use of `host` while volume deletion and do we really need this?